specifications/api/swagger.yaml

```diff
@@ -4660,6 +4766,33 @@ paths:
         - *parametersWalletId
       responses: *responsesListAssets
 
+    post:
+      operationId: mintBurnAssets
+      tags: ["Assets"]
+      summary: Mint/Burn
+      description: |
+        <p align="right">status: <strong>stable</strong></p>
+
+        Mint and burn assets from the wallet.
+
+        **⚠️ WARNING ⚠️**
+
+        Please note that due to the fact that there is no physical access to policy keys
```

Comment:
Kinda ... but the user can use cardano-address to derive the policy key themselves according to the forthcoming CIP.

We could also add an endpoint which returns policy keys ... in future.